model/config.go

```diff
@@ -2054,6 +2059,10 @@ func (ss *SqlSettings) isValid() *AppError {
 		return NewAppError("Config.IsValid", "model.config.is_valid.sql_idle.app_error", nil, "", http.StatusBadRequest)
 	}
 
+	if *ss.MaxLifetimeConns <= 0 {
```

Comment:
I think we should allow `0`, which effectively disabled the lifetime constraint.